org.eclipse.january.test/src/org/eclipse/january/dataset/DatasetUtilsTest.java

```diff
@@ -19,6 +24,41 @@
 
 public class DatasetUtilsTest {
 
+	@Test
+	public void testSplit() {
+		Dataset dataset = DatasetFactory.createFromObject(new Integer[] { 1, 2, 3 });
```

Comment:
It would demonstrate best practice by using primitive arrays instead of the boxed variety